pandas/tests/plotting/test_frame.py

```diff
@@ -231,14 +231,41 @@ def test_plot_xy(self):
     @pytest.mark.slow
     def test_logscales(self):
         df = DataFrame({'a': np.arange(100)}, index=np.arange(100))
+
         ax = df.plot(logy=True)
         self._check_ax_scales(ax, yaxis='log')
+        assert ax.get_yscale() == 'log'
+
+        ax = df.plot(logy='sym')
+        self._check_ax_scales(ax, yaxis='symlog')
+        assert ax.get_yscale() == 'symlog'
 
         ax = df.plot(logx=True)
         self._check_ax_scales(ax, xaxis='log')
+        assert ax.get_xscale() == 'log'
+
+        ax = df.plot(logx='sym')
+        self._check_ax_scales(ax, xaxis='symlog')
+        assert ax.get_xscale() == 'symlog'
 
         ax = df.plot(loglog=True)
         self._check_ax_scales(ax, xaxis='log', yaxis='log')
+        assert ax.get_xscale() == 'log'
+        assert ax.get_yscale() == 'log'
+
+        ax = df.plot(loglog='sym')
+        self._check_ax_scales(ax, xaxis='symlog', yaxis='symlog')
+        assert ax.get_xscale() == 'symlog'
+        assert ax.get_yscale() == 'symlog'
+
+    @pytest.mark.parametrize("input_param", ["logx", "logy", "loglog"])
+    def test_invalid_logscale(self, input_param):
+        # GH: 24867
+        df = DataFrame({'a': np.arange(100)}, index=np.arange(100))
+
+        msg = "Boolean, None and 'sym' are valid options, 'sm' is given."
+        with pytest.raises(ValueError, match=msg):
+            df.plot(**{input_param: "sm"})
```

Comment:
```suggestion
            df.plot(input_param=input_param})
```

Reply:
this doesn't seem what i meant... `input_param` is not an argument of `plot` function, `logx`, `logy`, `loglog` are.

Reply:
not sure what you mean by this comment, ``input_param`` will be substituted for the actual param

Reply:
ehh, i mean initially, i would like to assign an invalid input for `logx`, `logy` etc, and if we do `df.plot(input_param=input_param)` then, there seems no where to assign value?